frontend/public/components/storage/create-pvc.tsx

```diff
@@ -51,6 +52,21 @@ export class CreatePVCForm extends React.Component<CreatePVCFormProps, CreatePVC
       Gi: 'GiB',
       Ti: 'TiB',
     },
+    provisionerAccessModeMapping : {
```

Comment:
I'd add a link to the k8s documentation with the table of values

Reply:
This doesn't need to be stored in `state.` It's just a `const` variable. I'd add `readonly` too to avoid accidentally modifying it.

Reply:
ack